### PR TITLE
quote str kwarg values in `test_ops.py::TestCommon::test_jit_alias_remapping`

### LINKED ISSUE #57119: `test_ops.py::TestCommon::test_jit_alias_remapping` is broken for `str` kwargs
## 🐛 Bug

`test_ops.py::TestCommon::test_jit_alias_remapping` is broken for `str` kwargs:

https://github.com/pytorch/pytorch/blob/77721ee318d6785010144aa4569efb98199e7162/test/test_ops.py#L462-L467

In both cases the values are substituted in without modification. `str` values need to be quoted.

cc @gmagogsfm

### PR DESCRIPTION
**Pull request by pmeier**
Fixes #57119.

### CODE REVIEW

File: test/test_ops.py [hunk 1]
```diff
@@ -459,12 +459,19 @@ def test_jit_alias_remapping(self, device, dtype, op):
         # - args_kw is the string of args/kwargs used to call the op, same as args_annot_kw but
         # without type annotations
         args = ["t0"]
+
+        def parse_value(v):
```
Comment (vfdev-5) [inline test/test_ops.py:463]: nit: the name is well-chosen, I think.

**Review by vfdev-5 — COMMENTED** [review]
Looks good to me. A nit comment left.

File: test/test_ops.py (hunk 1 above)
Comment (pmeier) [reply test/test_ops.py:463]: You mean _not well-chosen_, correct? Do you have a better proposal? Maybe `process_value`?

Comment (vfdev-5) [reply test/test_ops.py:463]: Yes, not well-chosen. The methods adapts string values, something related with that I think should be in the name.

**Review by mruberry — APPROVED** [review]
Cool!

**Pull request closed without merging**